### DISCUSSION TITLE
[Docs] handler documentation can be modified to avoid a type error

### DISCUSSION THREAD

**Discussion by geminigeek** [nfriedly/express-rate-limit#410] [Ideas]
hi,

this is an excellent package , i just encountered a little issue , i think we can add this bit into handler docs for cases where we have message as function, to avoid type error like this

```
TypeError: options.message is not a function
```

```
  handler: (req, res, next, options) => {
     let message =
      typeof options.message === "function"
        ? options.message()
        : options.message
    return res.status(options.statusCode).send(message)
  },
```

**Comment by gamemaker1** [comment]
Hi,

The `handler` function you have written is the same as the default one, so you don't need to pass that in with the other options.

However, I do agree with you: it would be helpful to have this mentioned in the docs. Would you like to open a PR, changing the code snippet [here](https://github.com/express-rate-limit/express-rate-limit#handler) to your version?

Regards,
Vedant
